train.py

```diff
@@ -131,7 +131,7 @@ def _evaluate_acc_f1(self):
                     t_outputs_all = torch.cat((t_outputs_all, t_outputs), dim=0)
 
         test_acc = n_test_correct / n_test_total
-        f1 = metrics.f1_score(t_targets_all, torch.argmax(t_outputs_all, -1), labels=[0, 1, 2], average='macro')
+        f1 = metrics.f1_score(t_targets_all.cpu(), torch.argmax(t_outputs_all, -1).cpu(), labels=[0, 1, 2], average='macro')
```

Comment:
Do we have to explicitly call .cpu() to migrate to pytorch1.0 ?

Reply:
Yes, as pytorch1.0 does not allow tensors be operated by numpy methods when the tensor is located on GPU. And sklearn is based on numpy.